building/tracks/concept-exercises.md

```diff
@@ -34,25 +34,25 @@ A Concept Exercise has three types of files:
 
 These files are presented to the student to help explain the exercise.
 
-- `.docs/introduction.md`: introduce the concept(s) that the exercise teaches to the student (required)
-- `.docs/instructions.md`: provide instructions for the exercise (required)
-- `.docs/hints.md`: provide hints to a student to help them get themselves unstuck in an exercise (required)
+- `.docs/introduction.md`: introduces the concept(s) that the exercise teaches to the student (required)
+- `.docs/instructions.md`: provides instructions for the exercise (required)
+- `.docs/hints.md`: provides hints to a student to help them get themselves unstuck in an exercise (required)
```

Comment:
The file name that is indicated is indeed plural, and those files introduce the concept(s) that the exercise teaches.  Because it is not a single file, but many files with that same name, it is proper to treat it as "_they_ introduce the concept."